src/DependencyInjection/HttpCachePass.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace eLife\Journal\DependencyInjection;
+
+use PackageVersions\Versions;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+final class HttpCachePass implements CompilerPassInterface
+{
+    private static $packages = ['guzzlehttp/psr7', 'kevinrob/guzzle-cache-middleware'];
+
+    public function process(ContainerBuilder $container)
+    {
+        $definition = $container->findDefinition('cache.guzzle');
+
+        $tags = $definition->getTags();
+
+        $namespace = crc32(implode(',', array_map(Versions::class.'::getVersion', self::$packages)));
```

Comment:
Uniqueness is more important than readability here. Don't need to be accessed manually either.